Src/BootCamp.Chapter/Logging/TextLog.cs

```diff
@@ -3,18 +3,18 @@
 
 namespace BootCamp.Chapter.Logging
 {
-    class LogToText : Log
+    class TextLog : ILog
```

Comment:
ILog should be ILogger. Because it's not something that stores the text, but it's something that writes it.